Automattic/wp-calypso
# Security Settings: Fix for unsupported plans

#### Changes proposed in this Pull Request

* Use `security-settings` feature to determine whether to show the security settings page

#### Testing instructions



* Go to Settings > General for a free, personal, or premium Atomic site

* Without this change, you can access the Security page via the URL (/settings/security/SITE_URL). With this change, you should get an error page:

Before | After
------ | -----
<img width="756" alt="Screen Shot 2021-06-07 at 5 12 20 PM" src="https://user-images.githubusercontent.com/1689238/121088574-f34e4b00-c7b3-11eb-9208-611d649e7936.png"> | <img width="645" alt="Screen Shot 2021-06-07 at 5 11 30 PM" src="https://user-images.githubusercontent.com/1689238/121088582-f6493b80-c7b3-11eb-892b-f45b1aa9a979.png">

* Regression: you should still see the page on Jetpack sites and Atomic sites with supported plans. Simple sites should behave the same as free Atomic.
* Note that there will still be a Settings > Security nav item, that will be hidden by this Jetpack PR: https://github.com/Automattic/jetpack/pull/20384



Related to #51820
